docs/RELEASE.md

```diff
@@ -0,0 +1,36 @@
+# Vault Release Plan
+
+## Docker Image generation
+
+Docker images are hosted on [registry.scality.com](registry.scality.com).
+It has two names spaces for vault:
+
+* Production Namespace: registry.scality.com/vmware-ose-scality
+* Dev Namespace: registry.scality.com/vmware-ose-scality-dev
+
+Production namespace provides write access to a few admins and CI while read
+access is given to all the developers. Dev namespace provides write access
+only for the CI. The CI will push images with every CI build tagging the
+content with the developer’s branch short SHA-1 commit hash. This allows those
+images to be used by developers, CI builds, build chain and so on.
+
+## How to pull docker images
+
+```sh
+    docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    docker pull registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+```
```

Comment:
I might of wanted to mention this before now since you have made a few of these already... but unless we can get Harbot to show us the list of public tags ( eg https://registry.scality.com/harbor/projects/25/repositories/cloudserver%2Fcloudserver ) without authenticating it becomes a bit hard to use. Sure <tag> and <commit hash> are descriptive but unless you can see the actual list you are sort of having to play magic string bingo that the git tag just so happens to line up with our docker image. 
And yes, this should always line up but it doesn't mean it always will, a broken build or deleted tag on one side could always happen.

Obviously dev images are whatever as it is use at your own risk sort of thing which is fine ( although a 'latest-dev' would be nice to avoid looking it up )

Basically, we need a page like this https://hub.docker.com/_/mongo?tab=tags per public project.

Reply:
good question, indeed those are a bit painful, :thinking:

We're a bit outdated on harbor's version and may be missing (or not) some features towards that, I can have a look.

Reply:
You are able to search without logging in, but when you click a project it forces you to sign in. So, it is like half there...

Reply:
I checked on their new version, and saw nothing towards that, same behavior. We can raise the issue to the project though, see what the maintainers have to say.